features/step_definitions/assertions.rb

```diff
@@ -66,7 +66,7 @@
 end
 
 Then(/^the current directory will be a symlink to the release$/) do
-  run_vagrant_command(test_symlink_exists(TestApp.current_path))
+  run_vagrant_command(exists?("e", TestApp.current_path))
```

Comment:
This will make it check that the symlink points to a valid file/dir besides of just existing